packages/cozy-client/src/models/contact.js

```diff
@@ -247,3 +248,21 @@ export const getIndexByFamilyNameGivenNameEmailCozyUrl = contact => {
  * @returns {boolean}
  */
 export const isContact = doc => doc._type === CONTACTS_DOCTYPE
+
+/**
+ * @param {CozyClient} client - Instance of CozyClient
+ * @param {object} contact - A contact
+ * @returns {Promise<object>} - Modified or deleted contact
+ */
+export const deleteContact = (client, contact) => {
```

Comment:
The specific CRUD methods tend to be defined in the associated collection, so I feel like this belong more on ContactCollection. This way, the client don't have to know the `deleteContact` existence, it just has to call the high-level `client.destroy` method